src/software/ai/navigator/path_planner/path_planner_performance_test.cpp

```diff
@@ -0,0 +1,224 @@
+    obstacle_factory.createRobotObstacle({-2.5, 0}),
+    obstacle_factory.createRobotObstacle({-2.5, 0.5})};
+
+std::vector<PlannerTestCase> test_cases = {
+    {.name           = "Empty small area short path",
+     .start          = Point(-1, 0),
+     .end            = Point(1, 0),
+     .navigable_area = Rectangle({-2, -2}, {2, 2}),
+     .obstacles      = {},
+     .num_iterations = 10},
+
+    {.name           = "Empty medium area short path",
+     .start          = Point(-1, 0),
+     .end            = Point(1, 0),
+     .navigable_area = Rectangle({-3, -3}, {3, 3}),
+     .obstacles      = {},
+     .num_iterations = 10},
+
+    {.name           = "Empty divB area short path",
+     .start          = Point(-1, 0),
+     .end            = Point(1, 0),
+     .navigable_area = ::Test::TestUtil::createSSLDivBField().fieldBoundary(),
+     .obstacles      = {},
+     .num_iterations = 10},
+
+    {.name           = "Empty divB area long path",
+     .start          = Point(-4.5, 0),
+     .end            = Point(4.5, 0),
+     .navigable_area = ::Test::TestUtil::createSSLDivBField().fieldBoundary(),
+     .obstacles      = {},
+     .num_iterations = 10},
+
+    {.name           = "10 obstacles on full field short path",
+     .start          = Point(-2, 0),
+     .end            = Point(2, 0),
+     .navigable_area = ::Test::TestUtil::createSSLDivBField().fieldBoundary(),
+     .obstacles      = obstacles_10,
+     .num_iterations = 10},
+
+    {.name           = "20 obstacles on full field short path",
+     .start          = Point(-2, 0),
+     .end            = Point(2, 0),
+     .navigable_area = ::Test::TestUtil::createSSLDivBField().fieldBoundary(),
+     .obstacles      = obstacles_20,
+     .num_iterations = 10},
+
+    {.name           = "30 obstacles on full field short path",
+     .start          = Point(-2, 0),
+     .end            = Point(2, 0),
+     .navigable_area = ::Test::TestUtil::createSSLDivBField().fieldBoundary(),
+     .obstacles      = obstacles_30,
+     .num_iterations = 10},
+
+    {.name           = "10 obstacles on full field",
+     .start          = Point(-4.5, 0),
+     .end            = Point(4.5, 0),
+     .navigable_area = ::Test::TestUtil::createSSLDivBField().fieldBoundary(),
+     .obstacles      = obstacles_10,
+     .num_iterations = 10},
+
+    {.name           = "20 obstacles on full field",
+     .start          = Point(-4.5, 0),
+     .end            = Point(4.5, 0),
+     .navigable_area = ::Test::TestUtil::createSSLDivBField().fieldBoundary(),
+     .obstacles      = obstacles_20,
+     .num_iterations = 10},
+
+    {.name           = "30 obstacles on full field",
+     .start          = Point(-4.5, 0),
+     .end            = Point(4.5, 0),
+     .navigable_area = ::Test::TestUtil::createSSLDivBField().fieldBoundary(),
+     .obstacles      = obstacles_30,
+     .num_iterations = 10},
+};
+
+template <typename PlannerT>
+std::pair<std::string, PathPlannerConstructor> name_and_constructor()
```

Comment:
nit: function doesn't follow `camelCase` style guide

Reply:
changed to `camelCase`